Customize login system - work in progress

anaclair opened the pull request.

JakeHartnell commented:
Can you squash these in to a nicer commit? http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

JakeHartnell commented:
Next time we meet let's pair program on the rest of this! I rebased and merged your work into #12 and added some new things, so I am closing this branch in favor of that. This was a good start though! Thanks!

anaclair commented:
Pair programming would be great. 

Thanks for the update!

> On Nov 23, 2015, at 4:10 PM, Jake Hartnell <notifications@github.com> wrote:
> 
> Next time we meet let's pair program on the rest of this! I rebased and merged your work into #12 <https://github.com/CommonGarden/Meteor-IoT/pull/12> and added some new things, so I am closing this branch in favor of that. This was a good start though! Thanks!
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/CommonGarden/Meteor-IoT/pull/8#issuecomment-159110263>.
>
